src/request/providers/xhr.ts

```diff
@@ -205,6 +205,19 @@ export default function xhr(url: string, options: XhrRequestOptions = {}): Uploa
 	const task = <UploadObservableTask<XhrResponse>>new Task<XhrResponse>((resolve, reject) => {
 		timeoutReject = reject;
 
+		if (options.signal) {
+			options.signal.addEventListener('abort', () => {
+				let abortError: Error;
+				try {
+					abortError = new DOMException('Aborted', 'AbortError');
+				} catch {
+					abortError = new Error('Aborted');
+					abortError.name = 'AbortError';
+				}
+				reject(new DOMException('Aborted', 'AbortError'));
```

Comment:
Looks like you should use `abortError` here.

Reply:
Nice catch! Thanks!